app/Platform/Services/PlayerGameActivityService.php

```diff
@@ -153,8 +152,13 @@ public function initialize(User $user, Game $game, bool $withSubsets = false): v
                 'Achievements.Points',
                 'Achievements.BadgeName',
                 'Achievements.type',
-            ])
-            ->get();
+            ]);
+
+        if ($this->lastResetCreatedAt) {
+            $playerAchievementsQuery->where('player_achievements.unlocked_at', '>=', $this->lastResetCreatedAt);
+        }
```

Comment:
This should be redundant. If there are unlocks before the last reset, shouldn't we still process them?

Reply:
Removed this block in latest.